src/main/kotlin/uk/gov/justice/digital/hmpps/educationandworkplanapi/app/service/events/InboundEvents.kt

```diff
@@ -0,0 +1,68 @@
+package uk.gov.justice.digital.hmpps.educationandworkplanapi.app.service.events
```

Comment:
I'm not sure about the package structure here ..... 🤔 

If you look at what we already have, we have a `resource` package for everything 'REST API' related, along with child packages for mappers, validators etc. The REST stuff is all nicely self contained

If we extend that thought to messaging, I wonder if we need a package called `messaging` or similar as a sibling to `resource` with these types of classes in it, along with any child packages we might need? 🤔 

I know we already have a package called `service` .... I'm wondering if that is right (they feel like very different "services" to what you are adding here. We also have a `TimelineEventFactory` in that package, and I'm wondering if the terminology "event" might be overloaded and cause some confusion if its all in the same package structure

What do you think ? 🤔